README.md

```diff
@@ -1,75 +1,84 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Doctors Appointment Front-end
```

Comment:
- [ ] Your readme does not have a license section. Kindly add a license section on your readme.

## Tips and extra help

You already have a LICENSE file on the root of your project.  Just create a new "License" section and link to it in your readme.

That way, your app will be well documented 💪